layouts/shortcodes/issues-security.html

```diff
@@ -0,0 +1,19 @@
+<table class="security-cves">
+  <caption style="text-align:center">Table: Official Kubernetes CVE List</caption>
```

Comment:
This should be a localized string, really.

Reply:
Oh, also try to avoid putting CSS here. Instead, set a style rule that uses a `table.security-cves caption` selector.

Reply:
Updated